net/Socket.hpp

```diff
@@ -390,9 +390,11 @@ class Socket
 #if ENABLE_DEBUG
             if (std::getenv("COOL_ZERO_BUFFER_SIZE"))
             {
+#if !MOBILEAPP
```

Comment:
I wonder if we should instead stub out the functions that are causing the issue here... it looks like 8921e19d844da1f4f2c254edc66710120b5a6bfc and 3ba713aabf20279c5d83d039e5a7ac906b79c783 were specifically to try to remove MOBILEAPP directives, replacing them with Util::isMobileApp (also relevant could be 9afde4af160319b407b6ae82212eb84f9d3dac26 which adds the constexprs)

Reply:
I've reworked this pull request to make the functions stubs. Let me know if you see any additional changes that I should try.